code-challenges.js

```diff
@@ -10,7 +10,22 @@ var people = [
 ]
 // Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is a president of the galaxy." "Arthus Dent is a radio employee."
 
+const sentence = (array) => {
+  //First we use the .map function to run through each object in the array.
+  return array.map(value => {
+    //We then initialize a new variable to hold all the names and split them into a subarray of letters, and one for all the occupations.
+    let nameArr = value.name.split(" ")
+    let occArr = value.occupation
+    //Map through the subarray of characters and capitalize only the first character of each word, then attach the rest of the word starting from the letter after the first character to the capitalized letter.
+    nameArr = nameArr.map(value => {
+      return value.charAt(0).toUpperCase() + value.substring(1)
+    })
+    //The capitalized names and the occupation is then interpolated into a sentence, and then joined again so that it returns a combined string.
+    return `${nameArr.join(" ")} is a ${occArr}.`
+  }).join(" ")
+}
 
```

Comment:
Looks great! Logic is spot on. Just a small note, you could remove line 18 by just using value.occupation in your string interpolation on line 24.